cob_twist_controller/src/limiters/limiter.cpp

```diff
@@ -320,3 +356,40 @@ KDL::JntArray LimiterIndividualJointAccelerations::enforceLimits(const KDL::JntA
     return q_dot_norm;
 }
 /* END LimiterIndividualJointAccelerations **********************************************************************/
+
+/* BEGIN LimiterCartesianVelocities ********************************************************************/
+/**
+ * This implementation implements a saturation function to the Cartesian twists
+ */
+KDL::Twist LimiterCartesianVelocities::enforceCartesianLimits(const KDL::Twist& v_in) const
+{
+    KDL::Twist v_out(v_in);
+    double factor = 1.0;
+    // linear limts
+    for (unsigned int i = 0; i < 3; i++)
```

Comment:
allthough KDL::Twist provides this index access, I'd prefer you use the public members `vel` and `rot` to make it clear what is linear velocity and what is rotational velocity....e.g.
```
v_out.rot.x(copysign(std::min(limiter_params_.max_vel_lin, v_out.rot.x()), v_out.rot.x());
```